app/stats/knights.py

```diff
@@ -0,0 +1,63 @@
+from app.stats.armor import Armour
+from app.stats.weapon import Weapon
+from app.stats.potion import Potion
+
+
+class Knights:
+    knights = {}
+
+    def __init__(
+            self,
+            name: str,
+            power: int,
+            hp: int,
+            armour: list["Armour"] = None,
+            weapon: "Weapon" = None,
+            potion: "Potion" = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]